code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -1,137 +1,135 @@
 //Refer to life.dm for caller
-
 /mob/living/carbon/human/proc/handle_disabilities()
-	if(disabilities & ELECTROSENSE)
-		var/affect_chance = 100
-		var/affect_amount = 0
+	if(stat == DEAD)
+		return
+	if(disabilities & ELECTROSENSE && !dna.GetSEState(JAMSIGNALSBLOCK))
+		var/affect_chance = 30
+		var/affected = FALSE
 		if(head && istype(head,/obj/item/clothing/head/tinfoil))
-			affect_chance /= 2
+			affect_chance -= 15
```

Comment:
i'd rather you kept the divides, so the final result is 7.5 and not 1, which seems way too low. the lowering to thirty in base is okay

Reply:
1% is still pretty high, consider asthma attacks are 0.2%... 7.5% is greater than any other disability already, so the protection is basically useless

Reply:
the halloss is already debilitating enough that rye is necessary, whereas with most other disabilities, you can ignore